netsets/README.md

```diff
@@ -0,0 +1,6 @@
+# Netsets
+
+Netsets is a util package providing a map of "net.IPNet". It is used in various 
+placed in k8s.io/kubernetes to manage IP addresses for services and loadbalancers. 
+
+This package was originally copied over from `k8s.io/kubernetes/pkg/util/net/sets`.
```

Comment:
seems should change the `lib` parts in `https://github.com/kubernetes/utils/blob/master/README.md`, not add one :grin:

Reply:
ah, didn't see that there, good catch!

Reply:
fixed!